src/components/alert/alert.scss

```
@@ -68,11 +68,11 @@
   @include theme.styles {
     --theme-color-shadow-inner: #{sass-color.change(
         color.$brand-primary-darker,
-        $alpha: 0.2
+        $alpha: 0.4
       )};
     --theme-color-shadow-outer: #{sass-color.change(
         color.$brand-primary-darker,
-        $alpha: 0.1
+        $alpha: 0.3
```

Slight suggestion on just the outer shadow color, which feels a bit too indistinct from the darker outer shadow. 

```suggestion
        $alpha: 0.2
```